src/service/notification/notification.ts

```diff
@@ -33,21 +35,42 @@ export class Notification {
    */
   async getAll(body?: GetAllSubscriptionsQuery): Promise<ResponseDto<NotificationSubscription[]>> {
     return ErrorUtils.tryFail(async () => {
-      const subscriptions = await this.connector.get<AddressEventNotificationApi[]>({
+      const subscriptions = await this.connector.get<
+        [AddressEventNotificationApi | ContractAddressLogEventNotificationApi]
+      >({
         path: 'subscription',
         params: {
           pageSize: body?.pageSize?.toString() ?? '10',
           ...(body?.offset && { offset: body.offset.toString() }),
           ...(body?.address && { address: body.address }),
         },
       })
-      return subscriptions.map((notification) => ({
-        id: notification.id,
-        network: Utils.mapNotificationChainToNetwork(notification.attr.chain),
-        address: notification.attr.address,
-        url: notification.attr.url,
-        type: notification.type,
-      }))
+      return subscriptions.map((notification) => {
+        let result: Partial<NotificationSubscription> = {
```

Comment:
thinking if we cant have here const instead of let and instead of assigning we can directly returns